<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDicomViewer</class>
 <widget class="QMainWindow" name="QtDicomViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt DICOM Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowserLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>1251</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageViewer" name="ImageViewer1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>871</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="ScrollBarIV1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="ImageViewer" name="ImageViewer2" native="true">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageViewer" name="ImageViewer3" native="true">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>420</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinBoxIV1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>770</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="ScrollBarIV3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>770</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="ScrollBarIV2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>390</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open DICOM Series</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionOpenRegROIs">
   <property name="text">
    <string>导入轮廓线</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header>image_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
